src/BlazorIdentity/BlazorIdentity.csproj

```diff
@@ -6,4 +6,8 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Identity" Version="2.2.0" />
```

Comment:
You can't reference that package in this project. The point of the BlazorIdentity project is to be usable in all Blazor projects and that package is only for use in ASP.NET Core projects, and even then, only in unsupported versions, as that package was deprecated a long time ago. You need to add types in this project that shim/wrap the actual Identity types, in this case I think that's `UserLoginInfo`.